index.html

```diff
@@ -3025,7 +3114,7 @@ <h3>Using the Document Base as the Default Vocabulary</h3>
   </pre>
 </aside>
 
-<p>The <em>modified</em> key's value above is automatically type coerced to a
+<p>The <em>modified</em> key's value above is automatically type interpreted as a
```

Comment:
"type interpreted"? not familiar with that term? Did we rename "type coercion" to "type interpretation"?

Reply:
@iherman had some pushback on the notion of "type coercion", and "type interpretation" seemed like it might be a better fit. What happens in this case, is that the `@type` in the term definition is what allows the string value to be interpreted as a `dateTime`. But, it should probably be "interpreted as" rather than "type interpreted as".

Personally, I could go either way on this. Strictly speaking, "coercion" would be when something of one type is converted to another type, perhaps automatically because of an operation such as addition or concatenation. In this case, arguably, the string has no inherent type, and it is interpreted as an `xsd:string`, in the absence of any more explicit `@type` declaration.